l10n_br_account_payment_order/models/account_move_line.py

```diff
@@ -2,15 +2,65 @@
 #  Luis Felipe Miléo - mileo@kmee.com.br
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import models, fields, api
+from odoo import api, fields, models
+
+from ..constants import ESTADOS_CNAB, SITUACAO_PAGAMENTO
 
 
 class AccountMoveLine(models.Model):
     _inherit = 'account.move.line'
 
-    date_scheduled = fields.Date(string='Data Prevista')
+    date_scheduled = fields.Date(
+        string='Data Prevista',
+    )
+
```

Comment:
Esse campo nao esta usado, sera se realmente precisamos acrescentar ele? Nao poderia se usar o campo da data de pagamento planejado do account_payment_order que ja existe https://github.com/OCA/bank-payment/blob/12.0/account_payment_order/models/account_payment_order.py#L69 ?
Nao seria melhor acrescentar ele so o dia que for usado?